hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -2288,6 +2289,7 @@ private class Emit(
       mb.getArg(i)(typeToTypeInfo(pType)))
   }
 
+  // FIXME: is EmitRegion needed here?
```

Comment:
`er` doesn't appear to be used after this PR, but not sure if desired from interface standpoint

Reply:
also may be out of scope, just didn't want to lose track of this, since having unused args accumulate is probably not right

Reply:
defer this for now. Don't leave the fixme, though